.changelog/1631.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+core: Upgrade from unmaintained `boltdb/bolt` to `etcd-io/bbolt`
```

Comment:
I almost feel like a changelog entry isn't even required for this change. I usually avoid non-user-impacting changes from the CL.